libs/langgraph/langgraph/graph/state.py

```diff
@@ -702,11 +714,23 @@ def _get_channels(
         if name != "__slots__"
     }
     return (
-        {k: v for k, v in all_keys.items() if not is_managed_value(v)},
+        {k: v for k, v in all_keys.items() if isinstance(v, BaseChannel)},
```

Comment:
Why this change?

Reply:
Because mypy doesn't know that everything that isn't a managed value is a base channel